python/pyspark/taskcontext.py

```diff
@@ -162,7 +166,10 @@ def get(cls):
         running tasks.
 
         .. note:: Must be called on the worker, not the driver. Returns None if not initialized.
+            A RuntimeError will raise if it's not in a barrier stage.
```

Comment:
Can you avoid an abbreviation? `if it's not in a barrier stage` -> if it is not in a barrier stage`

Reply:
updated